### ISSUE TITLE
Unnecessary Removal of .NET Framework Support

### ISSUE THREAD

**Issue by joshsacks** [twilio/twilio-csharp#732]
### Issue Summary
Version 7.0.0 now requires a minimum version of .NET 6 and thus .NET Framework is no longer supported. However, there are still many, many Twilio customers like ourselves who - for a variety of reasons - are not able to transition our codebases from .NET Framework to Core.

The upshot is we're now excluded from using (and paying for) Twilio's new offerings/features made available through the helper library.

Moreover, reviewing the changes merged in between 6.18.0 and 7.0.0, there are no dependencies on the .NET 6 SDK that are not supported by .NET Framework.

**In other words, there is no technical reason to cut off Framework customers,** even if it's somewhat annoying to maintain the build configurations. (A small burden that a proudly "developer-first" company like Twilio should be willing to undertake.)

I, and surely many other customers, hope you will reconsider this decision to leave us in the cold.

**Comment by dsghi** [comment]
I appreciate them removing unnecessary dependencies and moving to modern framework support only. If you don't like it, fork it and modify it for your own needs. Let's keep the supply chain as clean as possible to reduce risks.

**Comment by joshsacks** [comment]
@dsghi Nice you like it - and your concern for others is telling - but the fact is the "supply chain" doesn't affect you one bit if you you're on a _modern framework_ like .NET 6*. NuGet will automagically reference the DLL compiled against .NET 6 and your code need not be sullied by even passing contact with the Framework version and its abominable "risks".

*.NET Framework is still supported and regularly patched by this company called Microsoft you may have heard of.

**Comment by smithery** [comment]
I don't appreciate them removing .NET Framework support. What happens when Twilio offers new endpoints or features to existing endpoints? They should have forked the .NET 6 version as a new library versus bricking support for the existing user base. I have a product with close to 100,000 lines of code using .NET Framework which cannot be ported overnight. I would love to move to .NET 6 but it is easier said than done.

**Comment by sbansla** [comment]
@joshsacks This will be fixed in upcoming release.
